src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -226,27 +226,27 @@ private bool ExecuteTermsOfUseCall(bool queryAcceptanceStatus)
             }, false);
         }
 
-        internal PackageUploadHandle PublishAsync(Package package, object files, IEnumerable<string> markdownFiles, bool isNewVersion, bool retainFolderStructure)
+        internal PackageUploadHandle PublishAsync(Package package, object files, IEnumerable<string> markdownFiles, bool isNewVersion, IEnumerable<string> roots, bool retainFolderStructure)
```

Comment:
I know this method is internal, but can you still add a comment, as this method is important, and it will help withj the changes in future.

Reply:
I added the comment with a bit more detailed information regarding the use of `roots` collection, hopefully this adds enough context.